decoder/expression_candidates.go

```diff
@@ -108,9 +140,10 @@ func constraintToCandidates(constraint schema.ExprConstraint, editRng hcl.Range)
 		})
 	case schema.MapExpr:
 		candidates = append(candidates, lang.Candidate{
-			Label:  fmt.Sprintf(`{ key =%s}`, labelForConstraints(c.Elem)),
-			Detail: c.FriendlyName(),
-			Kind:   lang.MapCandidateKind,
+			Label:       fmt.Sprintf(`{ key =%s}`, labelForConstraints(c.Elem)),
+			Detail:      c.FriendlyName(),
+			Description: c.Description,
```

Comment:
Small (unrelated) fix I made as I was reading through the code, probably doesn't affect much in practice though as we don't have too many map expressions with description.